### PR TITLE
Fix carousel display, item preview from HomeApp

### PR DESCRIPTION
**Pull request by thecalcc**
NHUB-475

## Checklist

- [x] This pull request is not adding new forms that use redux
- [x] This pull request is adding missing TypeScript types to modified code segments where it's easy to do so with confidence
- [x] This pull request is replacing `lodash.get` with optional chaining for modified code segments

### CODE REVIEW

**Comment by tomaskikutis** [comment]
does it look the same without overwriting CSS variables?

File: assets/home/components/HomeApp.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ import {personalizeHome} from 'agenda/actions';
 import {RadioButtonGroup} from 'features/sections/SectionSwitch';
 import {getCurrentUser} from 'company-admin/selectors';
 import {IPersonalizedDashboardsWithData} from 'home/reducers';
+import '@superdesk/common/dist/src/index.css';
```
Comment (tomaskikutis) [inline assets/home/components/HomeApp.tsx:25]: wasn't it already imported from base component meaning it wouldn't need additional imports?

Comment (thecalcc) [reply assets/home/components/HomeApp.tsx:25]: it is, but the HomeApp doesn't extend the base, therefor this import wouldn't be available in the homeapp if not explicitly imported as I did

Comment (tomaskikutis) [reply assets/home/components/HomeApp.tsx:25]: What else doesn't extend the base? If this is the only one then fine, but if there are others we should to come up with a way to import it once in the project

File: assets/am-news/index.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@ import {
     toggleNavigation,
     toggleNavigationById,
 } from '../search/actions';
-
+import {render} from 'render-utls';
```
Comment (tomaskikutis) [inline assets/am-news/index.ts:22]: should be `render-utils`?

File: assets/layout/components/SearchBase.tsx [hunk 3]
```diff
@@ -257,7 +275,7 @@ export default class BaseApp extends React.Component<any, any> {
     }
 }
 
-BaseApp.propTypes = {
+SearchBase.propTypes = {
```
Comment (tomaskikutis) [inline assets/layout/components/SearchBase.tsx:278]: if you copied info to IProps interface propTypes can be removed

**Review by tomaskikutis — APPROVED** [review]

**Pull request merged**